suse/factory.go

```diff
@@ -0,0 +1,238 @@
+package suse
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"regexp"
+	"strings"
+	"sync"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/libvuln/driver"
+	"github.com/quay/claircore/pkg/ovalutil"
+	"github.com/quay/zlog"
+	"golang.org/x/net/html"
+)
+
+//doc:url updater
+const base = `https://ftp.suse.com/pub/projects/security/oval/`
+
+var (
+	reELFile   = regexp.MustCompile(`suse.linux.enterprise.server.([1-9][1-9]).xml.gz`)
+	reLeapFile = regexp.MustCompile(`opensuse.leap.(1[0-9].[0-9]+).xml.gz`)
```

Comment:
does this add a lot more rows? Do you think we should go back to the bzip2 files and just accept SUSE 11 is no longer supported (or just make a workaround for it)?

Reply:
You can see from the PR desc that it does add some earlier opensuse LEAP versions. I added a minimum LEAP version, now the updates are only concerned with:
```
suse.linux.enterprise.server.11
suse.linux.enterprise.server.12
suse.linux.enterprise.server.15
opensuse.leap.15.5
opensuse.leap.15.6
```
The 3 supported ELS versions and the 2 [LEAP supported versions](https://en.opensuse.org/Lifetime).

Reply:
oh I didn't notice you edited it. I thought it was the same description as before 😂